getstarted.rst

```diff
@@ -5,73 +5,89 @@ Requirements
+We will be installing nio to a virtual environment. This keeps nio and all its dependencies isolated from other python projects and environements you may already have on your machine. You can read more about them `here <https://robinwinslow.co.uk/2013/12/26/python-3-4-virtual-environment/>`_.
+
+First, open a terminal in to the home directory. As a standard we will create a python virtual environement for each nio version.
+
+.. code-block:: bash
+
+    cd
+    mkdir nio
+    cd nio
+    mkdir projects
+    mkdir versions
+    cd versions
+    pyvenv-3.4 1.5.1
+
+Note: that last line will fail with Ubuntu 14.04 but can be fixed with `this <http://askubuntu.com/a/528625>`.
+
+You now need to activate your virtual environment. This will add a `(1.5.1)` to the beginning of your command line to indicate the virtual environemnt that you are using.
 
 .. code-block:: bash
 
-    $ mkdir nio
-    $ cd nio
-    $ virtualenv env
-    $ source env/bin/activate
+    source 1.5.1/bin/activate
 
-When you're done using nio, you can leave the virtual environment with `deactivate`. Be sure to always come back to this directory and run `source env/bin/activate` when you are going to be running nio.
+When you're done using nio, you can leave the virtual environment with `deactivate`. When using nio again, be sure to activate the virtual environemtn first with `source ~/nio/versions/1.5.1/bin/activate`.
 
-Download the latest nio and nioext wheels from the `nio Versions Page <http://n.io/versions>`_. Then install them using pip.
+OK, now we can finally install `nio <http://n.io/versions/nio/>`.
 
 .. code-block:: bash
 
-    $ wget http://n.io/versions/nio/nio-1.5.0-py3-none-any.whl
-    $ wget http://n.io/versions/nioext/nioext-1.5.0-py3-none-any.whl
-    $ pip install http://n.io/versions/nio/nio-1.5.0-py3-none-any.whl
-    $ pip install http://n.io/versions/nioext/nioext-1.5.0-py3-none-any.whl
+    pip install http://n.io/versions/nio/nio-1.5.1-py3-none-any.whl
+    pip install http://n.io/versions/nioext/nioext-1.5.1-py3-none-any.whl
 
 The installation of nio is now complete! You can run the instance from a project directory with the ``run_nio`` command. See :ref:`setting-up-a-project` for instructions on creating a project directory.
 
+We want to install one more thing before we move on though. We have devloped a Command Line Interface (CLI) to help manage your nio projects.
+
+.. code-block:: bash
+
+    pip install https://github.com/neutralio/nio-cli/archive/master.zip
+
 .. _setting-up-a-project:
 
 Setting up a NIO Project
---------------------
+------------------------
 
-If you already have an existing project configuration, you can skip this section.
+To execute this demo, you'll need `git` (a distributed version control tool) and a `GitHub account <http://github.com>` with `ssh access <https://help.github.com/articles/generating-ssh-keys/#platform-mac>`.
```

Comment:
This shouldn't point to "platform-mac". I would point to this link (it actually works on ALL platforms, and works better than other tutorials for windows

https://help.github.com/articles/generating-ssh-keys/#platform-all

The other option is to not specify any platform and it will auto-detect their platform

https://help.github.com/articles/generating-ssh-keys